api-docs/cloud-dedicated/v1-compatibility/swaggerV1Compat.yml

```
@@ -240,6 +240,83 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /ping:
+    get:
+      description: |
+        Reports the InfluxQL bridge querier health and the InfluxDB version of the instance.
+        
+        The response is a HTTP `204` status code to inform you the querier is available.
+
+        For InfluxDB Cloud Dedicated, this endpoint only checks the status of queriers; doesn't check the status of ingesters.
+
+        To check the health of ingesters before writing data, send a request to one of the [write endpoints](/influxdb/cloud-dedicated/api/v2/#tag/Write).
+
+        This endpoint doesn't require authentication.
```

At first, I thought this was referring to the write endpoint referenced above.

```suggestion
        The `/ping` endpoint doesn't require authentication.
```